src/schema/v2/homeView/sections/QuickLinks.ts

```diff
@@ -63,26 +63,23 @@ export const QUICK_LINKS: Array<NavigationPill> = [
     ownerType: OwnerType.collection,
     icon: undefined,
   },
-
-  // TODO: enable once navigation is sorted out, via minimumEigenVersion
-  // {
-  //   title: "Medium",
-  //   href: "/collections-by-category/Medium",
-  //   ownerType: OwnerType.collectionsCategory,
-  //   icon: "ArtworkIcon",
-  // },
+  {
+    title: "Medium",
+    href:
+      "/collections-by-category/Medium?homeViewSectionId=home-view-section-explore-by-category&entityID=Medium",
```

Comment:
Looks very disturbing 👀  I don't think this is the best approach, but it's working. Ideally we would have `/collections-by-category/Medium`, which we cannot use right now due to [the current setup on Eigen](https://github.com/artsy/eigen/blob/main/src/app/Scenes/CollectionsByCategory/Footer.tsx) where  `homeViewSectionId` and `entityID` are required to fetch the category data

Reply:
Yep, I agree. This coupling is on our radar to address/refactor but the href shown here should be a fairly low risk way to support the quick link in the meantime.